gm4_vecto_shamir/data/gm4_vecto_shamir/functions/init.mcfunction

```diff
@@ -0,0 +1,8 @@
+scoreboard objectives add gm4_ml_vecto_id dummy
```

Comment:
```suggestion
scoreboard objectives add gm4_vecto_id dummy
```
Shamirs should not use the `ml_` prefix. This change needs to be made in multiple sections of code throughout the module.